poker-pals/src/App.vue

```diff
@@ -1,13 +1,16 @@
 <template>
   <div id="app">
+    <Poker />
```

Comment:
Is there a reason why this is inside `App.vue` and not `Poker.vue`?

Reply:
the routing was not working on my end (I've got it to work now) but i still needed to see my stuff working so i put it there (and forgot to take it out). i am making changes now and will update this pull request